src/test/java/org/fcrepo/integration/auth/webac/WebACRecipesIT.java

```diff
@@ -163,7 +169,14 @@ public void scenario1() throws IOException {
 
         logger.debug("Can username 'smith123' read " + testObj);
         setAuth(request, "smith123");
-        assertEquals(HttpStatus.SC_OK, getStatus(request));
+        try (final CloseableHttpResponse response = execute(request)) {
+            assertEquals(HttpStatus.SC_OK, getStatus(response));
+
+            final Optional<String> header = Arrays.asList(response.getHeaders("Link")).stream().map(Header::getValue)
+                    .filter(aclLink::equals).findFirst();
+            assertTrue("Missing Link header", header.isPresent());
```

Comment:
@whikloj's code checks for a Link header that equals `aclLink` (note the part of the code with `.filter(aclLink::equals)`). The `rel` portion is part of that check and is defined above in line 164.